jormungandr/src/network/subscription.rs

```diff
@@ -290,7 +299,9 @@ impl Sink<net_data::Fragment> for FragmentProcessor {
     fn poll_flush(mut self: Pin<&mut Self>, cx: &mut Context<'_>) -> Poll<Result<(), Error>> {
         loop {
             if self.buffered_fragments.is_empty() {
-                self.poll_complete_refresh_stat(cx);
+                match self.poll_complete_refresh_stat(cx) {
+                    _ => (),
+                };
```

Comment:
I think now that this is slightly wrong. If the stats refresh future is not run to completion in some hypotetical task setup that does not `poll_flush` all the time regardless of input, the statistic may fall out of date. Wrap this call into `ready!`, it won't hurt.